docs/source/dataframe-best-practices.rst

```diff
@@ -236,14 +233,5 @@ the Apache Parquet format for Python:
    df1 = dd.read_parquet('path/to/my-results/', engine='fastparquet')
    df2 = dd.read_parquet('path/to/my-results/', engine='pyarrow')
 
-These libraries can be installed using:
 
-.. code-block:: shell
-
-   conda install fastparquet pyarrow -c conda-forge
-
-`fastparquet <https://github.com/dask/fastparquet/>`_ is a Python-based
-implementation that uses the `Numba <https://numba.pydata.org/>`_
-Python-to-LLVM compiler. PyArrow is part of the
-`Apache Arrow <https://arrow.apache.org/>`_ project and uses the `C++
-implementation of Apache Parquet <https://github.com/apache/parquet-cpp>`_.
+See :ref:`dataframe.parquet` for more on Dask Dataframe and Parquet.
```

Comment:
This is fine, but note the `:doc:` directive could also be used here to refer to the document by filename (no need to create an extra ref).